otx/algorithms/classification/tasks/inference.py

```diff
@@ -339,14 +353,19 @@ def _init_recipe(self):
         if train_type == TrainType.INCREMENTAL:
             recipe = os.path.join(recipe_root, "incremental.yaml")
 
+        if train_type == TrainType.SELFSUPERVISED:
+            recipe = os.path.join(recipe_root, "selfsl.yaml")
+            self.task_model_dir = os.path.join(self.task_model_dir, "selfsl")
+            self.task_pipeline_path = os.path.join(os.path.dirname(self.task_pipeline_path), "selfsl/data_pipeline.py")
```

Comment:
Is there any reason why this variable shouldn't use `data_pipeline_path` as it is?

Reply:
I reflected [this commit](https://github.com/openvinotoolkit/training_extensions/pull/1425/commits/68c7d2ebcd271e6e63c54b4e323216a6cd9d5521) and haven't reflected the recent commits for those variable names.
I'll update them at segmentation task.

Reply:
Yeah, when I went over this again, task_ is a little vague, because it can make a confusion like task_type (detection/classificaition...). I recommend to use model_dir and data_pipeline_path.